kyverno/assets/dashboards/overview.json

```diff
@@ -0,0 +1,1993 @@
+{
+    "author_name": "Datadog",
+    "description": "Kyverno is a Kubernetes-native policy engine that allows users to create, validate, and enforce policies for Kubernetes configurations. It enables automated policy management, including resource validation, mutation, and cleanup, to ensure compliance and best practices in Kubernetes clusters. This dashboard provides visibility into Kyverno's performance and operations, offering insights into policy execution, requeuing, dropping of elements, and overall controller health, helping administrators monitor and optimize policy enforcement.\n\n**Further reading:**\n\n- [Datadog Kyverno Integration Documentation](https://docs.datadoghq.com/integrations/kyverno/)\n- [Kyverno Docs](https://main.kyverno.io/docs/introduction/)\n- [Kyverno Monitoring Docs](https://main.kyverno.io/docs/monitoring/)\n",
```

Comment:
Just noting that the link to the Datadog docs doesn't work. But perhaps that's because the integration docs haven't been published yet?

Reply:
@drichards-87 Indeed. The integration isn't marked as public yet, so no publications has happened yet.